src/snap-confine.apparmor.in

```diff
@@ -94,19 +94,21 @@
 
     # for running snaps on classic
     mount options=(rw rslave) -> /,
```

Comment:
FYI: this is split because in the past the regular expression `/{tmp/snap.rootfs_*,}snap/` covered `/snap` which is no longer a constant.